XPlat/DropboxApiV2/nuget/Xamarin.Dropbox.Api.Core.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.Dropbox.Api.Core</id>
+    <title>Xamarin.Dropbox.Api.Core</title>
+    <version>4.5.3</version>
+    <authors>Xamarin</authors>
+    <owners>Xamarin</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Core library for Xamarin and dropbox Api v2 integration with Xamarin.Auth</description>
+    <copyright>Copyright © Xamarin. All Rights Reserved.</copyright>
+    <licenseUrl>https://components.xamarin.com/license/dropboxapiv2</licenseUrl>
+    <projectUrl>https://components.xamarin.com/dropboxapiv2</projectUrl>
+    <dependencies>
+      <group targetFramework="netstandard2.0">
+        <dependency id="Dropbox.Api" version="4.5.3" />
+        <dependency id="Xamarin.Auth" version="1.5.0.3" />
```

Comment:
Do we really need Auth?  Or is it highly recommended?

I am guessing that the dll can be used without Auth if the key was obtained other ways.

Reply:
yeah, we do for the help library as it provides the auth implementation